pkgs/tools/admin/zbctl/default.nix

```
@@ -0,0 +1,45 @@
+{ lib, stdenvNoCC, fetchurl, coreutils }:
```

I would recommend to space this out a bit more, it makes the diff easier to read if something is changed in the future. `coreutils` is also unused.
```suggestion
{ lib
, stdenvNoCC
, fetchurl
}:
```